README.md

```diff
@@ -2,3 +2,4 @@
 
 * `mortgage`: contains the notebook which runs ETL + ML on the Mortgage Dataset derived from [Fannie Mae’s Single-Family Loan Performance Data](http://www.fanniemae.com/portal/funding-the-market/data/loan-performance-data.html) ... download the mortgage dataset for use with the notebook [here](https://rapidsai.github.io/datasets/)
 * `utils`: contains a set of useful scripts for interacting with RAPIDS
+* `cuml`: contains four example notebooks showing the usage of different machine learning algorithms included in cuML: `knn`, `dbscan`, `pca` and `tsvd`. It also includes a small subset of the Mortgage Dataset used in the notebooks.
```

Comment:
Move this up above `mortgage` so that it's alphabetical.

`utils` should be the last thing listed.